pkg/names/names.go

```diff
@@ -31,3 +36,37 @@ const SERVICE_CA_CONFIGMAP = "openshift-service-ca"
 // MULTUS_VALIDATING_WEBHOOK is the name of the ValidatingWebhookConfiguration for multus-admission-controller
 // that is used in multus admission controller deployment
 const MULTUS_VALIDATING_WEBHOOK = "multus.openshift.io"
+
+// TRUST_BUNDLE_CONFIGMAP_KEY is the name of the data key containing
+// the PEM encoded trust bundle.
+const TRUST_BUNDLE_CONFIGMAP_KEY = "ca-bundle.crt"
+
+// TRUST_BUNDLE_CONFIGMAP is the name of the ConfigMap
+// containing the combined user/system trust bundle.
+const TRUST_BUNDLE_CONFIGMAP = "trusted-ca-bundle"
+
+// TRUST_BUNDLE_CONFIGMAP_NS is the namespace that hosts the
+// ADDL_TRUST_BUNDLE_CONFIGMAP and TRUST_BUNDLE_CONFIGMAP
+// ConfigMaps.
+const TRUST_BUNDLE_CONFIGMAP_NS = "openshift-config-managed"
+
+// TRUST_BUNDLE_CONFIGMAP_ANNOTATION is the name of the annotation that
+// determines whether or not to inject the combined ca certificate
+const TRUST_BUNDLE_CONFIGMAP_ANNOTATION = "config.openshift.io/inject-trusted-cabundle"
+
+// Proxy returns the namespaced name of the proxy
+// object named "cluster" in namespace "openshift-config-managed".
```

Comment:
proxy "cluster' resided in the default ns, not the "openshift-config-managed" ns. cc @squeed

Reply:
it doesn't reside in any namespace, it's a cluster scoped resource.

Reply:
Understood, this is a doc bug that was fixed in https://github.com/openshift/cluster-network-operator/pull/245/files#diff-dc24d687a587f957b553ce29a8dfaff8R40-R46. It will be fixed in this PR when I rebase.

Reply:
@bparees what is the difference? I thought default, cluster-scoped and non-namespaced are the same.

Reply:
`default` is an actual namespace you can go look at on a cluster

```
 oc get sa -n default
NAME       SECRETS   AGE
builder    2         124m
default    2         130m
deployer   2         124m

```

Reply:
@bparees thanks for the clarification.